src/AutoRest.CSharp.V3/Output/Models/Types/OutputLibrary.cs

```diff
@@ -45,16 +45,16 @@ private Client[] BuildClients()
 
         private Dictionary<Schema, ISchemaType> BuildModels()
         {
-            var allSchemas = (_codeModel.Schemas.Choices ?? Enumerable.Empty<ChoiceSchema>()).Cast<Schema>()
-                .Concat(_codeModel.Schemas.SealedChoices ?? Enumerable.Empty<SealedChoiceSchema>())
-                .Concat(_codeModel.Schemas.Objects ?? Enumerable.Empty<ObjectSchema>());
+            var allSchemas = _codeModel.Schemas.Choices.Cast<Schema>()
+                .Concat(_codeModel.Schemas.SealedChoices)
+                .Concat(_codeModel.Schemas.Objects);
 
             return allSchemas.ToDictionary(schema => schema, BuildModel);
         }
 
         private ISchemaType BuildModel(Schema schema) => schema switch
         {
-            SealedChoiceSchema sealedChoiceSchema => new EnumType(sealedChoiceSchema, _context),
+            SealedChoiceSchema sealedChoiceSchema => (ISchemaType)new EnumType(sealedChoiceSchema, _context),
```

Comment:
why did this change?

Reply:
It showed red squiggles in VS under the whole thing. It can't tell the type to use (it can't tell the common interface is `ISchemaType`). It compiled before (obviously), but I wonder why you didn't see the squiggles... Resharper, maybe?

Reply:
Not a big deal.